### DISCUSSION TITLE
Scope of discussions in this repository

### DISCUSSION THREAD

**Discussion by MikeMcC399** [corona-warn-app/cwa-website#3281] [General] [answered]
Should this discussion list be limited to topics concerning only the content of cwa-website or should it also cover topics related to other repositories?

Generally this repository is most active concerning the preparations for releases. It has not so far been labelled as the repository for the management of other repositories.

According to "About" of [cwa-documentation](https://github.com/corona-warn-app/cwa-documentation) the documentation repository contains content "Project overview".

**Comment by brianebeling** [comment, marked as answer]
Let's try to stick with website related information. Sometimes questions arise like #3285 that are not completely website related. These will be transferred then, in this case documentation might look like it's the most fitting. However, the user also mentioned the FAQ and CWA-Website. So we can extract information out of it and maybe update the website as well and it was decided to leave it in cwa-website. In this case it could also be helpful to mirror the answer to the cwa-documentation repo.
